examples/selected-executor-workflow.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
@sarabala1979 @simster7 this is a fully-filled out example with a test, description, and version.

Reply:
@jessesuen @terrytangyuan FYI